### PR TITLE
[Component with its own class name as service type] LPS-199657 Refactor Registry classes from object and content-dashboard module

### PR DESCRIPTION
**Pull request by Valmir-unicap**
- Ticket: https://liferay.atlassian.net/browse/LPS-199657

- Correction of PR: https://github.com/rafaprax/liferay-portal/pull/1375

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Review by MrJohn1911 — APPROVED** [review]

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContext.java [hunk 1]
```diff
@@ -73,12 +73,12 @@ public ContentDashboardAdminManagementToolbarDisplayContext(
 		AssetCategoryLocalService assetCategoryLocalService,
 		AssetVocabularyLocalService assetVocabularyLocalService,
 		ContentDashboardAdminDisplayContext contentDashboardAdminDisplayContext,
-		ContentDashboardItemFilterProviderRegistry
-			contentDashboardItemFilterProviderRegistry,
 		GroupLocalService groupLocalService,
 		HttpServletRequest httpServletRequest, ItemSelector itemSelector,
 		Language language, LiferayPortletRequest liferayPortletRequest,
 		LiferayPortletResponse liferayPortletResponse, Locale locale,
+		ServiceTrackerList<ContentDashboardItemFilterProvider>
```
Comment (rafaprax) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContext.java:80]: sort paramaters

File: modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContextTest.java [hunk 2]
```diff
@@ -227,19 +229,28 @@ public Portlet getPortlet() {
 			WorkflowConstants.STATUS_SCHEDULED
 		);
 
+		ServiceTrackerList<ContentDashboardItemFilterProvider>
```
Comment (rafaprax) [inline modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContextTest.java:155]: rename to `contentDashboardItemFilterProviderServiceTrackerList`

File: modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContextTest.java [hunk 3]
```diff
@@ -405,4 +251,79 @@ public boolean isShow(
 						"contentDashboardItemFilterParameterLabel: value2")));
 	}
 
+	private class ContentDashboardItemFilterProviderImpl
```
Comment (rafaprax) [inline modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminManagementToolbarDisplayContextTest.java:254]: rename to `MockContentDashboardItemFilterProvider `

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Pull request closed without merging**